weeks/week07/week07.md

```diff
@@ -1,3 +1,103 @@
 # Week 7: Web Automation with Playwright
 
-Coming soon.
+## Summary:
```

Comment:
Remove this heading. See how we are doing from other weeks to follow the same structure.